libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/list/InstructionListAdapter.java

```diff
@@ -72,7 +72,7 @@ public void onViewDetachedFromWindow(InstructionViewHolder holder) {
 
   public void updateSteps(Context context, RouteProgress routeProgress,
                           Locale locale, @NavigationUnitType.UnitType int unitType) {
-    if (distanceUtils == null || this.locale != locale || this.unitType != unitType) {
+    if (distanceUtils == null || this.locale.equals(locale) || this.unitType != unitType) {
```

Comment:
To match previous behavior (`this.locale != locale`), shouldn't be `!this.locale.equals(locale)` instead? Maybe it's a bug fix that I'm missing. Could you clarify?

Reply:
Ahh good catch 🍍